res/cmake/def.cmake

```diff
@@ -1,116 +1,42 @@
 
 
-if(testMPI)
-  if (NOT DEFINED PHARE_MPI_PROCS)
-    set(PHARE_MPI_PROCS 2)
-  endif()
-else()
-  if (NOT DEFINED PHARE_MPI_PROCS)
-    set(PHARE_MPI_PROCS 1)
-  endif()
-endif()
+set (PHARE_FLAGS ${PHARE_FLAGS})
+set (PHARE_WERROR_FLAGS ${PHARE_FLAGS} ${PHARE_WERROR_FLAGS})
+set (PHARE_PYTHONPATH "${CMAKE_BINARY_DIR}:${CMAKE_SOURCE_DIR}/pyphare")
 
 set (PHARE_HAS_HIGHFIVE "0")
 if (HighFive)
  set (PHARE_HAS_HIGHFIVE "1")
 endif()
 
-# Pybind errors with clang, it is default in GCC
-if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
-  set (CMAKE_CXX_FLAGS -fsized-deallocation)
+if (NOT DEFINED PHARE_MPI_PROCS)
+  set(PHARE_MPI_PROCS 1)
+  if(testMPI)
+    set(PHARE_MPI_PROCS 2)
+  endif()
 endif()
 
-if(coverage AND NOT MSVC)
-  set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pg -DHAVE_EXECINFO_H -g3")
+# Pybind errors with clang, it is default in GCC
```

Comment:
if that's for pybind shouldn't that rather be in the pybind config file?

Reply:
I'm not sure it's smart to separate setting variables like CMAKE_CXX_FLAGS all that much

Reply:
perhaps it should be "PHARE_FLAGS" instead of CMAKE_CXX_FLAGS